diff --git a/lib/treezor_client/models/payin.rb b/lib/treezor_client/models/payin.rb
index 9324101..3eec42b 100644
--- a/lib/treezor_client/models/payin.rb
+++ b/lib/treezor_client/models/payin.rb
@@ -388,6 +388,8 @@ module TreezorClient
 
       if attributes.has_key?(:'ibanBic')
         self.iban_bic = attributes[:'ibanBic']
+      elsif attributes.has_key?(:'ibanBIC')
+        self.iban_bic = attributes[:'ibanBIC']
       end
 
       if attributes.has_key?(:'ibanTxEndToEndId')
